### DISCUSSION TITLE
how to use pagination in bagisto 2.0 instead of load more on category page

### DISCUSSION THREAD

**Discussion by NasirHayat** [bagisto/bagisto#9498] [General]
I would really appreciate if if someone guides me i want replace loadmore button on category pages with pagination next 1 2 3 prev

**Comment by devansh-webkul** [comment]
Hi @NasirHayat,

Are you referring to Shop's category page?

**Reply by NasirHayat** [reply]
yes category / search result pages

**Reply by NasirHayat** [reply]
at front end and thanks alot for your reply

**Reply by devansh-webkul** [reply]
You just need to override this view,
https://github.com/bagisto/bagisto/blob/master/packages/Webkul/Shop/src/Resources/views/categories/view.blade.php

And then if you see the Ajax request you will see the meta key and in this meta, you will have all the pagination-related details which you can add.

We are simply using the paginate method provided by Laravel, the rest you can do as per your need.

**Reply by NasirHayat** [reply]
i have seen this file https://github.com/bagisto/bagisto/blob/master/packages/Webkul/Shop/src/Resources/views/partials/pagination.blade.php can we not use this file ?

**Reply by NasirHayat** [reply]
![Edit-Import](https://github.com/bagisto/bagisto/assets/3838274/49d3c00c-1cba-4040-9100-f19b668358ea)
![Screenshot 2024-02-20 160333](https://github.com/bagisto/bagisto/assets/3838274/102efaa5-8d60-4472-84e2-035fccfd9ba6)
![Screenshot 2024-02-20 160437](https://github.com/bagisto/bagisto/assets/3838274/3ad4fdca-aae3-4155-a310-e88b8d171bc0)
i am trying to user new data transfer product import working fine except images m i missing something or there some bug

**Reply by NasirHayat** [reply]
@devansh-webkul  cloud you please guide on data import why images not begin uploaded

**Reply by jitendra-webkul** [reply]
Images should be placed inside "storage/app/import/product-images" as mentioned in the first screenshot. But you have uploaded images in wrong directory "storage/product-images"

**Reply by NasirHayat** [reply]
@jitendra-webkul  jitendra-webkul
![Screenshot 2024-02-21 122144](https://github.com/bagisto/bagisto/assets/3838274/05b00f08-f19f-4f20-b684-9b4db54dfe54)
i put images there but still its not working. one question it supports only webp format or other images formats too?

**Reply by jitendra-webkul** [reply]
The functionality is currently designed to operate exclusively for new products; if the product already exists, image uploads won't be initiated. However, we have plans to include this option in a forthcoming release.

**Reply by jitendra-webkul** [reply]
The system is compatible with various formats, and upon upload, images will be automatically converted to WebP.
